services/hooks/src/main.js

```diff
@@ -45,6 +45,7 @@ const load = loader({
       serviceName: 'hooks',
       monitor: monitor.childMonitor('db'),
       statementTimeout: process === 'server' ? 30000 : 0,
+      azureCryptoKey: cfg.azure.cryptoKey,
       dbCryptoKeys: cfg.postgres.dbCryptoKeys,
```

Comment:
These have `dbCryptoKeys` already and I think the major changelog  mentions adding `dbCryptoKeys`. Do we still need `azureCryptoKey`?

Reply:
The idea was to continue allowing azureCryptoKey in addition to dbCryptoKeys, rather than requiring deployers to change that setting at the same time they deploy the new version. Otherwise, we should remove the ignored configuration parameter.  As it stands, the service will start up with AZURE_CRYPTO_KEY set, but fail to encrypt/decrypt, leaving deployers wondering what they've done wrong (and they haven't done anything wrong).

Reply:
> As it stands, the service will start up with AZURE_CRYPTO_KEY set, but fail to encrypt/decrypt

Why is that?

Reply:
As it stands (meaning: without this patch), the value of AZURE_CRYPTO_KEY is never used.

Reply:
Ah, ok. Gotcha. Thanks!

Reply:
If and when we do our deployment-config-cleanup, we can remove the AZURE_CRYPTO_KEY config from all services at once.